docs/configuration.rst

```diff
@@ -163,6 +163,16 @@ Toolbar options
   but want to render your application in French, you would set this to
   ``"en-us"`` and :setting:`LANGUAGE_CODE` to ``"fr"``.
 
+.. _UPDATE_ON_FETCH:
+
+* ``UPDATE_ON_FETCH``
+
+  Default: ``False``
+
+  This controls whether the toolbar should update to the latest AJAX
+  request when it occurs. This is useful to enable if when "boosting"
+  the web application with a JavaScript library such as htmx.
```

Comment:
I like the changelog entry a lot. "to control whether the toolbar automatically updates to the latest AJAX request or not" reads beautifully. 

The second sentence is a bit strange because of the "if when". Maybe something like this would be better? "This is especially useful when using htmx boosting or similar JavaScript techniques."

I'm not a native speaker/writer either though, so I'm not sure.

Reply:
"if when" is definitely a mistake :sweat_smile: Your English is perfect.